apps/emqx_connector/test/emqx_connector_api_SUITE.erl

```diff
@@ -718,6 +718,36 @@ t_create_with_bad_name(Config) ->
     ?assertMatch(#{<<"kind">> := <<"validation_error">>}, Msg),
     ok.
 
+%% Checks that we correctly handle `throw({bad_ssl_config, _})' from
+%% `emqx_connector:convert_certs' and massage the error message accordingly.
+t_create_with_bad_tls_files(Config) ->
+    ConnectorName = atom_to_binary(?FUNCTION_NAME),
+    Conf0 = ?KAFKA_CONNECTOR(ConnectorName),
+    Conf = Conf0#{<<"ssl">> => #{<<"cacertfile">> => <<"bad_pem_file">>}},
+    ?check_trace(
+        begin
+            {ok, 400, #{
+                <<"message">> := Msg0
+            }} = request_json(
+                post,
+                uri(["connectors"]),
+                Conf,
+                Config
+            ),
+            ?assertMatch(
+                #{
+                    <<"file_read">> := <<"enoent">>,
+                    <<"pem_check">> := <<"invalid_pem">>,
+                    <<"which_options">> := [[<<"cacertfile">>]]
+                },
```

Comment:
Nit. Might be out-of-scope but I definitely feel this needs to be made easier to understand, currently it sounds cryptic IMO.